### ISSUE TITLE
repurpose members of respLike.part

### ISSUE THREAD

**Issue by pe-ro** [music-encoding/music-encoding#1316]
Creating a separate issue for the suggestion I made in the discussion of issue #555 where I said that the members of respLike.part (`<arranger>`, `<author>`, `<composer>`, `<contributor>`, `<editor>`, `<funder>`, `<librettist>`, `<lyricist>`, and `<sponsor>`) or their replacement (`<agent>`) can be made more biographical; that is, they can be used as containers for more detailed information that just that normally used in bibliographic description, such as those defined by TEI -- affiliation, age education, faith, dates of activity, gender, language knowledge, nationality, occupation, persona, residence, sex, socio-economic status, states (time-based characteristics), and traits (unalterable characteristics).  This list can also be extended to include other characteristics like place of birth and death.

**Comment by pe-ro** [comment]
Going down the TEI `<person>` route may be reinventing the wheel.  Another option for recording biographical/authority data is to put it in `<extMeta>` in an already standard form. Metadata Authority Description Schema (MADS) seems to be a good choice for this, but other options exist.

**Comment by bwbohl** [comment]
I don't think we should model non-[edit]bibliographic data in MEI, there are other formats (those mentioned) that can be used for such purposes.

**Comment by pe-ro** [comment]
@bwbohl, I presume you meant to say that we shouldn't model **_non-bibliographic_** data in MEI.  We're already well into modeling bibliographic data.
